nuxeo-retention/pom.xml

```diff
@@ -10,6 +10,10 @@
   <name>Nuxeo Retention Management Core</name>
 
   <dependencies>
+    <dependency>
+      <groupId>jakarta.el</groupId>
+      <artifactId>jakarta.el-api</artifactId>
+    </dependency>
```

Comment:
@kevinleturc we had 
```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/fox/workspace-2025/nuxeo-retention/nuxeo-retention/src/main/java/org/nuxeo/retention/service/RetentionManagerImpl.java:[328,34] cannot access jakarta.el.ELException
```
after https://github.com/nuxeo/nuxeo-lts/commit/2e0c84757cad39362911e5fa2c697cf40c06334e

This is what you explained in https://github.com/nuxeo/nuxeo-lts/pull/2422#issue-2597043459 ? Right?

Reply:
Yes